Source/CheckoutAPIClient.swift

```diff
@@ -107,12 +108,48 @@ public class CheckoutAPIClient {
                 }
         }
     }
+    
+    /// Create a card token
+    ///
+    /// - parameter card: Card used to create the token
+    /// - parameter completion: Callback to execute if the request is successful or failed
+    public func createCardToken(
+        card: CkoCardTokenRequest,
+        completion: @escaping ((Swift.Result<CkoCardTokenResponse, NetworkError>) -> Void)
+    ) {
+        let url = "\(environment.urlPaymentApi)\(Endpoint.tokens.rawValue)"
+        let jsonEncoder = JSONEncoder()
+        // swiftlint:disable:next force_try
+        var urlRequest = try! URLRequest(url: URL(string: url)!, method: HTTPMethod.post, headers: headers)
```

Comment:
one more : I remember you fixed all these force unwraps in other PR. Not sure why are they not reflecting here ? . Can you please check.